dataplanes/vpp/yaml/vpp-daemonset.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+spec:
+  template:
+    metadata:
+      labels:
+        app: nsm-vpp-dataplane
+    spec:
+      hostNetwork: true
+      hostIPC: true
+      hostPID: true
+      containers:
+        - name: vpp-daemon
+          securityContext:
+            privileged: true
+          image: networkservicemesh/vpp-daemon:latest
+          imagePullPolicy: Always
+          volumeMounts:
+          - name: run
+            mountPath: /var/run
+          - name: lib
+            mountPath: /var/lib
+          - name: lib-modules
+            mountPath: /lib/modules
+            readOnly: true
+        - name: vpp
+          securityContext:
+            privileged: true
+          image: networkservicemesh/vpp:latest
+          imagePullPolicy: Always
+          volumeMounts:
+          - name: run
+            mountPath: /var/run
+          - name: lib
+            mountPath: /var/lib
+          - name: lib-modules
+            mountPath: /lib/modules
+            readOnly: true
+          - name: hugepages
```

Comment:
Does this mean we need to run Kubernetes with the FEATURE_GATE flag for HugePages enabled, per [this page](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)? Also, I assume we can still run this on VMs, right?

Reply:
huge pages are now supported without any feature gate flags. But you are right, VM where this runs MUST have huge pages and kubelet should be capable to discover them.